crates/tracing/src/lib.rs

```diff
@@ -15,6 +15,7 @@ use wasmcloud_core::OtelConfig;
 
 #[cfg(feature = "otel")]
 pub mod context;
+pub mod http;
```

Comment:
Can we put this behind `#[cfg(feature = "otel")]` as well since it uses OTEL libraries?